docs/jdbc-mysql.md

```diff
@@ -0,0 +1,77 @@
+# Connecting to MySQL using JDBC
```

Comment:
When we reshuffle the modules, I think we should revisit keeping these in the docs folder or with the module.